opencti-platform/opencti-graphql/src/migrations/1687438290000-threat-actor-rename.js

```diff
@@ -0,0 +1,34 @@
+import { elUpdateByQueryForMigration } from '../database/engine';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+import { DatabaseError } from '../config/errors';
+import { convertTypeToStixType } from '../database/stix-converter';
+
+const entityTypeChange = (fromType, toType, indices) => {
+  const updateQuery = {
+    script: {
+      params: { toType, prefix: convertTypeToStixType(toType) },
+      source: 'ctx._source.entity_type = params.toType;',
+    },
+    query: {
+      bool: {
+        must: [
+          { term: { 'entity_type.keyword': { value: fromType } } },
```

Comment:
This will only migrate entities of type "Threat-Actor", how about relationships and history ? for relationships, we have fromType, toType

Reply:
Also EntitySetting exist with "target_type": "Threat-Actor",

Reply:
@SouadHadjiat You were right, my migration wasn't okay. It was missing the fromType and the target_type, it should be fix by now :)

Reply:
what about `toType` ?